README.MD

```diff
@@ -79,6 +100,21 @@ strictly part of DAFT but rather DAFT includes tools and libraries to interact
 with the device so own test can be written or existing test suites can be
 integrated.
 
+![](readme_pictures/daft_run.png)
+
+_Running daft._
+
+1. Start daft by executing the following command in the PC Host: `daft DUT PATH/TO/IMG.img`. Replace DUT with the desired DUT e.g. `minnowboard` or `joule` and `PATH/TO/IMG.img` with the image you want to flash. You can also run the command directly in BBB but it then it won't lock the DUTs. Running daft in BBB is done with the command `aft DUT PATH/TO/IMG.img`.In case you want to only flash or test or boot the DUT refer to [DAFT commandline interface](#32-daft-commandline-interface) to add correct options for DAFT.
+2. DAFT then checks if the device is blacklisted in the lockfiles. If not, then it locks it so other simultaneous DAFT runs won't use the same DUT.
+3. BBB reboots the DUT and proceeds to start the support image in the DUT. The support image is needed so that the target image can be flashed to the SDcard or to another memory storage. BBB emulates itself as multiple components to enable automated and simple function:
+   1. As a keyboard (to get through DUT bios).
+   2. As a usbdrive (so that the DUT can boot the support image).
+   3. As a ethernet (to give the DUT an IPaddress).
```

Comment:
IP address* You could also generalize that ethernet is needed so DUT can be connected to as the ethernet is needed so BBB can run commands on it (needed for flashing and testing) and also so the DUT gets the image to flash.